guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -10,6 +10,24 @@ Select the option that corresponds with operating system and version you want to
 
 == [[repositories-rhel-8]]{RHEL} 8
 
+For configuring repositories in {EL} 8, you must enable `ruby:2.7` and `postgresql:12`.
+
+. Verify the module versions:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf module list --enabled
+----
++
+If the version of modules is incompatible:
```

Comment:
If the version of modules *are* incompatible.

Reply:
> It is referring to `version` and not `modules`. slightly_smiling_face

My mistake! Now looking back at it, I see it. :)

Reply:
> My mistake! Now looking back at it, I see it. :)

No problem @bangelic Do have a look at my other PRs :heart: :+1: